app/src/components/domain/RiskImminentEvents/Pdc/index.tsx

```diff
@@ -20,6 +35,22 @@ import {
 import EventDetails from './EventDetails';
 import EventListItem from './EventListItem';
 
+function getAlertType(alertType: 'WARNING' | 'WATCH' | 'ADVISORY' | 'INFORMATION') {
+    if (alertType === 'WARNING') {
+        return 'Red';
+    }
+    if (alertType === 'WATCH') {
+        return 'Orange';
+    }
+    if (alertType === 'ADVISORY') {
+        return 'Green';
+    }
+    if (alertType === 'INFORMATION') {
+        return 'Blue';
+    }
+    return undefined;
```

Comment:
It's better we use the same alert types and defining a specific PDC Layer based on it.